app/controllers/user_controller.rb

```diff
@@ -22,7 +22,28 @@ def show
 
     set_show_requests if @show_requests
 
+    @private_requests = []
```

Comment:
tiny nitpick - typo of "requests" in the extended commit message for 78fa5bd3a2!

Reply:
Will try to remember to reword in the squash :)